documentation/src/test/java/org/hibernate/validator/referenceguide/chapter08/BootstrappingTest.java

```diff
@@ -65,6 +67,16 @@ public void traversableResolver() {
 		//end::traversableResolver[]
 	}
 
+	public void clockProvider() {
+		//tag::clockProvider[]
+		ValidatorFactory validatorFactory = Validation.byDefaultProvider()
+				.configure()
+				.clockProvider( new FixedClockProvider( ZonedDateTime.now() ) )
```

Comment:
Might make more sense to actually show usage of a fixed date: `ZonedDateTime.of("2016-...")`.